### PR TITLE
HODS-402 | update handler configs and code

### PR DESCRIPTION
**Pull request by Luka-Abey**

### CODE REVIEW

File: app/uk/gov/hmrc/individualsbenefitsandcreditsapi/services/ScopesHelper.scala [hunk 1]
```diff
@@ -20,47 +20,89 @@ import play.api.hal.Hal.{linksSeq, state}
...
 import java.util.UUID
 import javax.inject.Inject
 
 class ScopesHelper @Inject()(scopesService: ScopesService) {
+  /**
+   * @param scopes The list of scopes associated with the user
+   * @param endpoint The endpoint for which to construct the query string
+   * @return A google fields-style query string with the fields determined by the provided endpoint and scopes
+   */
+//  def getQueryStringFor(scopes: Iterable[String], endpoint: String): String = {
```
Comment (matthewharris1) [inline app/uk/gov/hmrc/individualsbenefitsandcreditsapi/services/ScopesHelper.scala]: Remove comments

**Review by matthewharris1 — APPROVED** [review]
1 comment mate

**Review by matthewharris1 — CHANGES_REQUESTED** [review]
Might of clicked approve by mistake ...

**Review by ajchilts — APPROVED** [review]

**Comment by ajchilts** [comment]
@parkeradam  - approved pending addressing comment from @matthewharris1 .

**Pull request merged**